packages/wallet/test/hardware/TrezorKeyAgent.test.ts

```diff
@@ -49,6 +62,35 @@ describe('TrezorKeyAgent', () => {
     expect(typeof keyAgent.extendedAccountPublicKey).toBe('string');
   });
 
+  test('sign and submit tx', async () => {
+    const outputs = [
+      {
+        address: Cardano.Address(
+          'addr_test1qpu5vlrf4xkxv2qpwngf6cjhtw542ayty80v8dyr49rf5ewvxwdrt70qlcpeeagscasafhffqsxy36t90ldv06wqrk2qum8x5w'
+        ),
+        value: { coins: 11_111_111n }
+      },
+      {
+        address: Cardano.Address(
+          'addr_test1qz2fxv2umyhttkxyxp8x0dlpdt3k6cwng5pxj3jhsydzer3jcu5d8ps7zex2k2xt3uqxgjqnnj83ws8lhrn648jjxtwq2ytjqp'
+        ),
+        value: {
+          assets: new Map([[AssetId.TSLA, 6n]]),
+          coins: 5n
+        }
+      }
+    ];
+    const props = {
+      outputs: new Set<Cardano.TxOut>(outputs)
+    };
+    const txInternals = await wallet.initializeTx(props);
+    const tx = await wallet.finalizeTx(txInternals);
+    expect(tx.body).toBe(txInternals.body);
+    expect(tx.id).toBe(txInternals.hash);
+    expect(tx.witness.signatures.size).toBe(1);
+    await expect(wallet.submitTx(tx)).resolves.not.toThrow();
```

Comment:
Using a mock tx submit provider this is probably not asserting anything useful

Reply:
ok